float64 finger_0
float64 finger_2
float64 finger_4
